apis/meta/v1alpha1/converters_test.go

```diff
@@ -0,0 +1,32 @@
+package v1alpha1
+
+import (
+	"reflect"
+	"testing"
+
+	fuzz "github.com/google/gofuzz"
+	"github.com/sensu/sensu-go/internal/apis/meta"
+)
+
+func Test_convert_ObjectMeta_To_meta_ObjectMeta(t *testing.T) {
```

Comment:
This actually also tests `convert_meta_ObjectMeta_to_ObjectMeta()`, doesn't it?
I would suggest a name that reflects that fact, because I had to read the whole test to figure that out  and not add a comment about missing tests for `meta.ObjectMeta` -> `ObjectMeta` :(